dip_server/dip_platform/api/game/game_api.py

```diff
@@ -0,0 +1,40 @@
+from dip_platform.api.generic_view import GenericView
+from dip_platform.management.game import game_management
+from dip_platform.management.game import player_management
+
+
+class GameView(GenericView):
+    def handle_get(self, request, game_id):
+        game = game_management.get_game_by_id(game_id)
+        game_players = player_management.get_players_for_game(game_id)
+        player_units = {
+            player['id']: player_management.get_units_for_player(player['id'])
+            for player in game_players
+        }
+        player_territories = {
+            player['id']: player_management.get_territories_for_player(player['id'])
+            for player in game_players
+        }
+
+        status = 200
+        payload = {
+            'id': game['id'],
+            'players': [{
+                'id':   player['id'],
+                'name': player['name'],
+
+                'units': [{
+                    'id':            unit['id'],
+                    'type':          unit['type'],
+                    'territoryName': unit['territory_name'],
+                } for unit in player_units[player['id']] ],
+
+                'ownedTerritories': [{
+                    'id':            owned_territory['id'],
+                    'territoryName': owned_territory['territory_name']
```

Comment:
I don't actually know what I am talking about, so this could already be done somewhere and I just can't read python well. I suspect you can find a way to do the conversion from camelCase to snake_case automatically. My first guess was to add it to `_build_response`. I don't actually think this should prevent you from merging this commit though.